api/functions/src/notifications/notificationService.ts

```diff
@@ -9,12 +9,16 @@ if (process.env.NODE_ENV == 'development') {
   sgMail.setApiKey(process.env.SENDGRID_API_KEY!);
 }
 
+// Use this to instantly receive an email
+// notifyUsers(1, 1.1, 'KJPGkuNVivC5scheOIz0', '2hKLutzjE0ufQen8xTm3'); 
+
 export async function notifyUsers(
   threshold,
   recordedValue,
   sensorId,
   machineId
 ) {
+  console.log('notifyUsers function is being called');
```

Comment:
should probably remove this log statement right

Reply:
Removed